packages/reactotron-server/src/server/apollo/resolvers/commands.ts

```diff
@@ -0,0 +1,26 @@
+import { Resolver, Query, Subscription, Root, Args, Arg } from "type-graphql"
+
+import { MessageTypes } from "../../messaging"
+import { Command } from "../../schema"
+import { commands } from "../../datastore"
+import { CommandAddedArgs } from "../../schema/command"
+
+@Resolver()
+export class CommandsResolver {
+  @Query(() => [Command])
+  commands(@Arg("clientId", { nullable: true }) clientId?: string) {
+    if (clientId) {
+      return commands.byClientId(clientId)
+    }
+
+    return commands.all()
+  }
+
+  @Subscription(() => Command, {
```

Comment:
This is really starting to look like C#.  😆

Reply:
I am not complaining about that.